src/main/java/no/nav/veilarboppfolging/controller/v2/ManuellStatusV2Controller.java

```diff
@@ -0,0 +1,97 @@
+package no.nav.veilarboppfolging.controller.v2;
+
+import lombok.RequiredArgsConstructor;
+import no.nav.common.types.identer.Fnr;
+import no.nav.veilarboppfolging.client.dkif.DkifKontaktinfo;
+import no.nav.veilarboppfolging.controller.request.VeilederBegrunnelseDTO;
+import no.nav.veilarboppfolging.controller.v2.response.ManuellStatusV2Response;
+import no.nav.veilarboppfolging.controller.v2.response.ManuellV2Response;
+import no.nav.veilarboppfolging.repository.enums.KodeverkBruker;
+import no.nav.veilarboppfolging.service.AuthService;
+import no.nav.veilarboppfolging.service.ManuellStatusService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+@RestController
+@RequestMapping("/api/v2/manuell")
+@RequiredArgsConstructor
+public class ManuellStatusV2Controller {
+
+    private final AuthService authService;
+
+    private final ManuellStatusService manuellStatusService;
+
+    // TODO: Sjekk om vi trenger begge endepunktene. Dvs / og /status
+
+    @GetMapping
+    public ManuellV2Response hentErUnderManuellOppfolging(@RequestParam("fnr") Fnr fnr) {
+        authService.skalVereInternBruker();
+        authService.sjekkLesetilgangMedFnr(fnr);
+
+        boolean erManuell = manuellStatusService.erManuell(fnr);
+
+        return new ManuellV2Response(erManuell);
+    }
+
+    @GetMapping("/status")
+    public ManuellStatusV2Response hentManuellStatus(@RequestParam("fnr") Fnr fnr) {
+        authService.skalVereInternBruker();
```

Comment:
srvveilarbaktivitet trenger tilgang til denne informasjonen (KRR reservasjon status + under manuell oppfølging) som systembruker for en kafka-tjeneste.